### ISSUE TITLE
Graticule rotation of projview

### ISSUE THREAD

**Issue by bhhsieh** [healpy/healpy#724]
Hi, I am using projview and I want to rotate the map together with the graticule just like what mollview does.
But it seems that projview doesn't rotate the graticule automatically.
Are there any way to do it?

```
hp.mollview(m,rot=(theta,phi,0))
hp.graticule(color='w')

projview(
    m,
    rot=(theta,phi,0),
    graticule=True,
    graticule_labels=True,
    unit="cbar label",
    xlabel="longitude",
    ylabel="latitude",
    cb_orientation="vertical",
    projection_type="mollweide",
    title="Mollweide projection, astro convention (default)",    
)
```
![截圖 2021-10-23 07 05 12](https://user-images.githubusercontent.com/34892184/138528042-707a8b5d-449f-47d5-b542-7a2719f7b5c3.png)

**Comment by zonca** [comment]
@F-Tomas would you like to take a look at this issue?

**Comment by F-Tomas** [comment]
> @F-Tomas would you like to take a look at this issue?

Yes, I will look at it.

**Comment by F-Tomas** [comment]
I believe that this issue can be also closed.

**Comment by zonca** [comment]
Closed in #751
